roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -58,6 +58,9 @@ spec:
               name: "cluster"
           env:
             -
+              name: "DC_NAME"
+              value: "{{deploy_name}}"
```

Comment:
Addendum to my comment, we generate this name so its known before hand.  This pattern will allow us to also be backwards compatible.  I don't know if downward api supports escaped characters at all and I'm not certain when the CLI introduced support either.  Possibly 1.5?